rewrite-java/src/main/java/org/openrewrite/java/JavaPrinter.java

```diff
@@ -794,7 +794,6 @@ public J visitVariableDeclarations(VariableDeclarations multiVariable, PrintOutp
         }
         if (multiVariable.getVarargs() != null) {
             visitSpace(multiVariable.getVarargs(), Space.Location.VARARGS, p);
-            p.append("...");
```

Comment:
This doesn't look right. It seems like without this the test would ONLY pass if the "..." were inside the whitespace of `multiVariable.getVarargs()`, which isn't correct either. No "..." should exist within the whitespace of _any_ element, except within a comment.

Reply:
making the change here because the  "..." has been included in `multiVariable.getVarargs()` and don't want to print it twice.
I thought `Space VariableDeclarations.varargs`  ([code](https://github.com/openrewrite/rewrite/blob/main/rewrite-java/src/main/java/org/openrewrite/java/tree/J.java#L5464)) is intentionally to store "...", it's a Space type, so maybe we should put "..." into `Space.comments`? what do you think?

Reply:
I thought about this again, and it seems like putting "..." to the `Space.comments ` does not look correct because it's not a comment.

Reply:
Changed this line back, and instead, remove "..." from `multiVariable.getVarargs()`